monai/transforms/croppad/array.py

```diff
@@ -819,13 +815,12 @@ def threshold_at_one(x):
 
     """
 
-    @deprecated_arg_default("allow_smaller", old_default=True, new_default=False, since="1.2", replaced="1.3")
```

Comment:
this is change of default was introduced after tagging v1.2, I think this warning should be kept for a few more versions

Reply:
Sure, how about replacing it after "1.5"?